app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -272,7 +272,7 @@ def test_check_telemetry_sticky_targeting(
     pref_name = "sticky.targeting.test.pref"
 
     requests.delete("http://ping-server:5000/pings")
-    targeting = helpers.load_targeting_configs()[-1]
+    targeting = helpers.load_targeting_configs()[0]
```

Comment:
I agree but no one should be removing the test targeting constant and we should not allow that to be removed wither. This will always be first. https://github.com/mozilla/experimenter/blob/main/app/experimenter/targeting/constants.py

Reply:
Regardless of whether someone does or not you can't rely on the ordering of that list.  We change the order of lists as they appear in the frontend all the time, if you know the slug of the targeting config you want to use in the test it should be explicitly declared.  Plus it's not obvious from the way this is written that we are using a specific targeting config, having it explicitly stated makes it clear what the test depends on.

Reply:
Okay, ill just remove this step since the default slug is fine.